index.html

```diff
@@ -0,0 +1,48 @@
+            <li><span>1937</span> - Finishes university and takes a job in the US Forestry Service</li>
+            <li><span>1938</span> - Marries wife of 69 years Margret Gibson. Gets laid off due to budget cuts. Inspired by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about breeding pest-resistent plants.</li>
+            <li><span>1941</span> - Tries to enroll in the military after the Pearl Harbor attack, but is rejected. Instead, the military asked his lab to work on waterproof glue, DDT to control malaria, disinfectants, and other applied science.</li>
+            <li><span>1942</span> - Receives a Ph.D. in Genetics and Plant Pathology</li>
+            <li><span>1944</span> - Rejects a 100% salary increase from Dupont, leaves behind his pregnant wife, and flies to Mexico to head a new plant pathology program. Over the next 16 years, his team breeds 6,000 different strains of disease resistent wheat - including different varieties for each major climate on Earth.</li>
+            <li><span>1945</span> - Discovers a way to grown wheat twice each season, doubling wheat yields</li>
+            <li><span>1953</span> - crosses a short, sturdy dwarf breed of wheat with a high-yeidling American breed, creating a strain that responds well to fertilizer. It goes on to provide 95% of Mexico's wheat.</li>
+            <li><span>1962</span> - Visits Delhi and brings his high-yielding strains of wheat to the Indian subcontinent in time to help mitigate mass starvation due to a rapidly expanding population</li>
+            <li><span>1970</span> - receives the Nobel Peace Prize</li>
+            <li><span>1983</span> - helps seven African countries dramatically increase their maize and sorghum yields</li>
+            <li><span>1984</span> - becomes a distinguished professor at Texas A&M University</li>
+            <li><span>2005</span> - states "we will have to double the world food supply by 2050." Argues that genetically modified crops are the only way we can meet the demand, as we run out of arable land. Says that GM crops are not inherently dangerous because "we've been genetically modifying plants and animals for a long time. Long before we called it science, people were selecting the best breeds."</li>
+            <li><span>2009 </span> - dies at the age of 95.</li>
+        </ul>
+        <div class="comment">
+            <p>"Borlaug's life and achievement are testimony to the far-reaching contribution that one man's towering intellect, persistence and scientific vision can make to human peace and progress."</p>
+            <p>-- Indian Prime Minister Manmohan Singh</p>
+        </div>
+        <div class="final">
+            <p>If you have time, you should read more about this incredible human being on his <a id="tribute-link" href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank">Wikipedia entry.</a></p>
+        </div>
+    </div>
+    <div class="footer" ></div>
+</body>
+</html>
```

Comment:
instead of using <div> tags repeatedly try to use HTML5 semantic elements like <header> to include your page title and main paragrah then use <main> to include all your site articles and sections also you can use <section> tag inside your <tag> to encapsoluate all required information for cetain section inside your site for example your main image caption and commpent can be inside <section> histrorical events can be in another <section> and so on .
if you have a footer use <footer> tag all these new tags are very important for the screen readers to detect your site correctly to help people who has disabilities to not miss any important information inside your site